src/main/kotlin/no/nav/helse/spenn/oppdrag/dao/OppdragState.kt

```diff
@@ -16,6 +17,7 @@ enum class OppdragStateStatus {
     STARTET,
     SIMULERING_OK,
     SIMULERING_FEIL,
+    STOPPET,
```

Comment:
STOPPET burde flyttes helt ned etter FEIL. for å beholde ordinal verdien. Ikke så viktig her, men ved senere endring i enumeration.

Reply:
ikke sant... men kanskje bare beholde den der den er nå nå da? For state-machine-flytmessig ligger den jo der før sending til OS :)